.github/workflows/deploy-integration-example-app.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       - name: Example package end to end test
         run: |
           cd workbench-core/example/infrastructure
-          node ../../../common/scripts/install-run-rushx.js cypress run --verbose
```

Comment:
This caused the failure ?

Reply:
Yes, cypress run doesn't have parameter `--verbose`.